content/js3/blocks/fetching-data/index.md

```diff
@@ -0,0 +1,62 @@
++++
+title = '🐕 Fetching data'
+headless = true
+time = 20
+facilitation = false
+emoji= '🧩'
+[objectives]
+    1='Update a problem statement using web APIs to fetch data'
++++
+
+So far we have displayed films data stored in our JavaScript code. But real applications fetch data from servers over the internet. We can restate our problem as follows:
+
+> _Given_ a **web API that serves** film data
```

Comment:
It seems like we're using "web API" to mean both "JS APIs browsers tend to implement" (as per https://developer.mozilla.org/en-US/docs/Web/API) and "APIs served over the web by other servers".

I'm not sure this is useful - should we maybe use another term to mean the latter?

Reply:
Ah yes I think this is a mismatch between me and Mitch.